content/why/contents.lr

```diff
@@ -0,0 +1,11 @@
+_hidden: yes
+---
+content:
+
+Slova Nova was initiated by members of a house-church movement in Columbus, Ohio, called Xenos. We believe God gave Xenos a special strength: a robust equipping and discipleship ministry that has nurtured thousands of house-church workers – men and women of character able to teach, lead, and disciple.
+
+But while English-speaking communities like Xenos have a wealth of equipping, Christians in some parts of the world report a lack of biblical resources for house-church workers – the exact type of resources Slova Nova seeks to translate and digitally distribute.
+
+Slova Nova is an effort to partner with indigenous house-church leaders worldwide by providing resources that they identify as valuable to their ministry. And our hope is that this will lead to healthier house-church planting movements around the world, to the glory of God.
```

Comment:
Remember, this is a `markdown` field.  It is possible to use markup here if you want.